### DISCUSSION TITLE
Welcome to fantasy_football_chat_bot Discussions!

### DISCUSSION THREAD

**Discussion by dtcarls** [dtcarls/fantasy_football_chat_bot#90] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by danigle** [comment]
The Readme mentions "For troubleshooting, join the discord", but the link doesn't work.

**Reply by dtcarls** [reply]
Fixed the link.
